engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/modelconfig/ModelConfigToLoad.scala

```diff
@@ -0,0 +1,17 @@
+package pl.touk.nussknacker.engine.modelconfig
+
+import com.typesafe.config.Config
+
+/**
+  * Used by a running process on Flink to recreate full process config using only config passed during process
+  * deployment.
+  *
+  * This class with config is serialized in Flink graph so we separate it from the rest of configs to keep it
+  * small (see e.g. FlinkProcessCompiler), as it's only part that has to be passed to Flink.
+  */
+case class ModelConfigToLoad(inputConfig: Config, modelConfigLoader: ModelConfigLoader) {
```

Comment:
Why this class still exists? Why not to `loadConfig` directly in `ModelData`? Because of backward compatibility or there are some other (serialization?) reasons?

Reply:
Yes, we can get rid of it. It was left because of serialization reasons though I see now that serialVersionUID has been changed. Anyway I need to check if a Flink job can start from a previous savepoint. If not then the only way not to break compatiblity would be leaving `ModelConfigToLoad` as is and removing `ModelConfigLoader.resolveConfigDuringExecution` method.